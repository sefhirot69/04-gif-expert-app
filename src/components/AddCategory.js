import React, {useState} from 'react';
import PropTypes from 'prop-types';

AddCategory.propTypes = {

};

function AddCategory(props) {

    const [inputValue, setInputValue] = useState('Hola mundo');

    const handleInputChange = (e) => {
        setInputValue(e.target.value)
    };

    const handleSubmit = (e) => {
        e.preventDefault();

        console.log('Submit hecho');
    };

    return (
        <>
            <form onSubmit={handleSubmit}>
                <input
                    type="text"
                    value={inputValue}
                    onChange={handleInputChange}
                />
            </form>
        </>
    );
}

export default AddCategory;