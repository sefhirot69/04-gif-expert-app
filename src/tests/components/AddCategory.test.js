import {shallow} from "enzyme";
import AddCategory from "../../components/AddCategory";

describe('Pruebas sobre AddCategory', () => {

    const setCategories = () => {};
    const wrapper = shallow(<AddCategory setCategories={setCategories}/>)

    test('Debe mostrar el componente AddCategory', () => {

        expect(wrapper).toMatchSnapshot();
    });
});