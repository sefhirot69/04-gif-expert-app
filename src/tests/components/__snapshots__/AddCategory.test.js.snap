// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre AddCategory Debe mostrar el componente AddCategory 1`] = `
<Fragment>
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      type="text"
      value=""
    />
  </form>
</Fragment>
`;
